python/astropy.rst

```diff
@@ -13,7 +13,7 @@ Integration of Astropy core into the LSST software stack is an ongoing process t
 This document is not discussing Astropy affiliated packages, use of which must go through the standard :doc:`RFC </communications/rfc>` process.
 Not all Astropy core packages can be used by default.
 
-The following packages can be used internally in packages if they do not leak into public APIs:
+The following packages can be used internally in packages:
```

Comment:
I think `time` and `units` are now accepted enough that they could even be in public APIs, not just internal.  But I'm not sure we want the rest in APIs.